src/app/components/node/node.component.ts

```diff
@@ -130,102 +104,119 @@ export class NodeComponent implements OnInit, OnChanges, OnDestroy {
-      totals['t1_cgc_genes'] += parent_data.jsonData?.t1_cgc_genes || 0;
-      totals['genes'] += parent_data.genes;
-      totals['drugs'] += parent_data.jsonData?.drugs || 0;
-    }
-    this.parentsAggregate = totals;
+    // build variant info per consequence
+    const consequenceToVariantInfo = new Map<string, VariantInfo>();
+    clusterIds.forEach(clusterId => {
+      const cluster = this.clusterIdToCluster.get(clusterId);
+      cluster?.variants.forEach(variant => {
+        const snp = this.snvIdToDisplayVariant.get(variant);
+        if (snp) {
+          const consequence = snp.consequence ?? '';
+          if (!consequenceToVariantInfo.has(consequence)) {
+            consequenceToVariantInfo.set(consequence, getEmptyVariantInfo());
+          }
+          const variantInfo = consequenceToVariantInfo.get(consequence)!;
+          variantInfo.mutationsAll.add(variant);
+          const geneName = snp.gene;
+          if (geneName && geneName.length > 0) {
+            variantInfo.genesAll.add(geneName);
+            if (snp.cgcGeneInfo) {
+              variantInfo.genesCgc.add(geneName);
+              if (snp.cgcGeneInfo.tier === 1) {
+                variantInfo.mutationsTier1.add(variant);
+              }
+            }
+          }
+          // TODO drugs
+        } else {
+          console.warn('Could not find SNP for variant ' + variant);
+        }
+      });
+    });
+    // consolidate variant info across consequences
+    this.consolidatedVariantInfo = combineVariantInfos([...consequenceToVariantInfo.values()]);
+    // sort for table
+    this.sortedSummaryTableRows = [...consequenceToVariantInfo.entries()]
+      .map(([consequence, variantInfo]) => ({ consequence, variantInfo }))
+      .sort((a, b) => b.variantInfo.mutationsAll.size - a.variantInfo.mutationsAll.size);
   }
 
   updateStyle(): void {
+    const startColor = this.displayNode!.color;
     //get children color (assume first is the same as all of them)
-    const startColor = this.color || '#555';
-    const childrenColor = (this.children && this.children[0]?.color) || startColor;
+    const childrenColor = (this.displayNode?.children && this.displayNode.children[0]?.color) || startColor;
     let styles = {
       '--cell_block_color': startColor,
       '--gradient_start': startColor,
       '--gradient_end': childrenColor
     } as any;
-    //only apply the height for the final tree end
-    if (this.parentTotal) {
-      const hasSelectedBlock = this.selectedBlocks.length > 0;
-      const selectedBlockLevel = hasSelectedBlock ? this.selectedBlocks[0].blockLevel : 0;
-      styles['flex'] = (!this.ignoreScaling && hasSelectedBlock && selectedBlockLevel >= this.blockLevel && !this.isSelected) ? '1' : '' + (this.getProportion() * 1000); //proper value is 100, using 1000 to force this to shrink super small when selected
+    //only apply the height for the final tree end // TODO FIXME not what I'm doing at the moment; see next line
```

Comment:
I was having problems getting the block sizes right so I cheated and set them on each block for the time being. at some point we can take another look at it (or maybe it'll be obvious to you, Chad!)